dojo/tools/twistlock/parser.py

```diff
@@ -3,55 +3,39 @@
 import json
 import logging
 import hashlib
+import textwrap
 
-from django.utils.html import escape
 from dojo.models import Finding
 
 logger = logging.getLogger(__name__)
 
 
 class TwistlockCSVParser(object):
 
-    def get_field_from_row_or_default(self, row, column, default_value):
-        field = row[column]
-        if field is None or field == '':
-            return default_value
-        return escape(field)
-
     def parse_issue(self, row, test):
 
         if not row:
             return None
 
-        id_column = 4
-        line_column = 5
-        code_line_column = 6
-        vulnerability_id_column = 10
-        severity_column = 13
-        package_name_column = 14
-        package_version_column = 16
-        cvss_column = 18
-        fix_status_column = 19
-        description_column = 20
-        data_vulnerability_id = self.get_field_from_row_or_default(row, vulnerability_id_column, '')
-        data_package_version = self.get_field_from_row_or_default(row, package_version_column, '')
-        data_fix_status = self.get_field_from_row_or_default(row, fix_status_column, '')
-        data_package_name = self.get_field_from_row_or_default(row, package_name_column, '')
-        data_id = self.get_field_from_row_or_default(row, id_column, '')
-        data_severity = self.get_field_from_row_or_default(row, severity_column, 'Info').capitalize()
-        data_cvss = self.get_field_from_row_or_default(row, cvss_column, '')
-        data_description = self.get_field_from_row_or_default(row, description_column, '')
+        data_vulnerability_id = row['CVE ID']
+        data_package_version = row['Package Version']
+        data_fix_status = row['Fix Status']
+        data_package_name = row['Packages']
+        data_id = row['Id']
+        data_severity = row['Severity'].capitalize()
```

Comment:
`capitalize` is useless IMHO. See my comment on creating a function to manage severity values below.

Reply:
Fixed in the conversation above.